lib/orbit_web/controllers/sign_in_export_controller.ex

```diff
@@ -31,6 +31,18 @@ defmodule OrbitWeb.SignInExportController do
     end
   end
 
+  @spec override_expiration_date([map()], String.t()) :: String.t()
+  def override_expiration_date(override_list, type) do
+    {:ok, datetime, _} =
+      Enum.find(override_list, fn cert -> cert["type"] == type end)["expires"]
+      |> DateTime.from_iso8601()
+
+    datetime
+    |> DateTime.shift_zone!("America/New_York")
+    |> DateTime.to_date()
+    |> Date.to_iso8601()
```

Comment:
Heads up- this is a little messy because the `expires` is stored in the jsonb column as an ISO timestamp, but I want to provide Excel users with an Excel-compatible date in `YYYY-MM-DD`. (Yes, this is lossy)

Reply:
Should we always work with expiration dates as Dates / `YYYY-MM-DD` strings instead of converting them to datetimes then? Or at least save them in the database like that?

I think the date better represents the data. Especially since in datetimes, "Jan 13" is usually represented as `2025-01-13 00:00:00`, but expiration dates usually mean expiring at the end of the day, so at `2025-01-13 23:59:59`, so the datetime isn't even the right datetime.